notebooks/254-llm-chatbot/254-rag-chatbot.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<p>I think in future, we can have more embeddings models, it maybe not good choice to differ them only by language (e.g. having names). I think it is better to convert LLM, then embeddings model and add choice of Chinese if model support Chinese?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
done